pyproject.toml

```diff
@@ -28,6 +28,10 @@ tensorflow = [
     "keras",
 ]
 torch = [
-    "torch==1.13.1+cu117",
+    "torch",
+    "torchaudio",
```

Comment:
Do you why we need torchaudio?

Reply:
It is what [the torch install page](https://pytorch.org/get-started/locally/) recommends so I left `torchaudio` in even though I agree that it is quite likely that we don't need it.  We may not need `torchvision` either; we don't reference it explicitly in our code.  I will run the code without either of them and see what happens.

Reply:
I cargo-cult add it to torch projects, too.

Reply:
I removed `torchaudio` but kept `torchvision`.